docs/src/explanations/chisel-enum.md

```diff
@@ -0,0 +1,131 @@
+# ChiselEnum
+
+The ChiselEnum type is an easy way to get away from value encodings and reduce the chance of error when encoding muxes, opcodes, and functional unit operations. As opposed to the Chisel type Enums, ChiselEnums allow for names to be explicitly passed through Bundles like IO. 
+
+## Importing the ChiselEnum module
+
+```scala mdoc
+import chisel3._
+import chisel3.util._
+import chisel3.stage.ChiselStage
+import chisel3.experimental.ChiselEnum
+
+```
+
+## Functionality and Examples
+
```

Comment:
```suggestion

## Functionality and Examples

```scala mdoc
// Imports used in the following examples
import chisel3._
import chisel3.util._
import chisel3.stage.ChiselStage
import chisel3.experimental.ChiselEnum
```

```

Reply:
I had the import in separate ticks from the example. You removed the end ticks from "Importing the ChiselEnum module" but not the start ticks of the next block

Reply:
yeah but i think that is an artifact of github's suggest changes API... anyway, consider this just as a suggestion